python/ray/serve/backend_worker.py

```diff
@@ -71,32 +70,26 @@ def __init__(self, name, _callable, is_function):
         self.callable = _callable
         self.is_function = is_function
 
-        self.error_counter = 0
-        self.latency_list = []
-
-    def get_metrics(self):
-        # Make a copy of the latency list and clear current list
-        latency_list = self.latency_list[:]
-        self.latency_list = []
-
-        return {
-            "{}_error_counter".format(self.name): {
-                "value": self.error_counter,
-                "type": "counter",
-            },
-            "{}_latency_s".format(self.name): {
-                "value": latency_list,
-                "type": "list",
-            },
-        }
+        self.metric_client = MetricClient.connect_from_serve(
```

Comment:
Can we also add a counter for the number of times each replica has been started? This would help monitor if workers are crashing.